installation/openhabian.md

```diff
@@ -73,32 +73,63 @@ Learn more about the Raspberry Pi as your platform for openHAB and about the req
 * Enjoy! 🎉
 
 * The device will be available under it's IP or via the local DNS name `openhabianpi`
 * [Connect to the openHAB 2 portal](http://docs.openhab.org/configuration/packages.html) (available after a few more minutes): [http://openhabianpi:8080](http://openhabianpi:8080)
 * [Connect to the Samba network shares](http://docs.openhab.org/installation/linux.html#mounting-locally) with `openhab:habopen`
-* If you encounter a connection problem: [See here](#faq-successful)
+* If you encounter any setup problem, [please continue here](#faq-successful)
 
 You can stop reading now.
 openHABian has installed and configured your openHAB system and you can start to use it right away.
-If you want to get in touch with the system or install one of the previously mentioned optional components, you can come back here later.
+If you want to get in touch with the system or want to install one of the previously mentioned optional components, you can come back here later.
 
 Ready for more?
-[Connect to your Raspberry Pi SSH console](https://www.raspberrypi.org/documentation/remote-access/ssh/windows.md)  using the username and password `pi:raspberry`.
+[Connect to your Raspberry Pi SSH console](https://www.raspberrypi.org/documentation/remote-access/ssh/windows.md) using the username and password `pi:raspberry`.
 You will see the following welcome screen:
 
 ![openHABian login screen](images/openHABian-SSH-MotD.png)
 
 ➜ Continue at the ["First Steps"](#first-steps) chapter below!
 
-### Pine64
+### Pine A64
 
-***... Coming Soon! ...***
+We are also providing a ready-to-use system image for the Pine A64.
+The image is based on the official [Ubuntu Base Image by longsleep](http://wiki.pine64.org/index.php/Pine_A64_Software_Release) and other than the Raspberry Pi image comes as a compressed 4GB file.
+After bootup, the latest version of openHAB 2 and the mentioned settings and tools are installed.
+All packages will be downloaded in their newest version and configured to work without further modifications.
+
+Learn more about the Pine A64 as your platform for openHAB and about the requirements over in our [Pine64 article](pine.html).
+
+**Setup:**
+
+* [Download the latest "openHABianPine64" SD card image file](https://github.com/ThomDietrich/openhabian/releases) (note: the file is *xz* compressed)
+* Extract the compressed archive with your favorite program
+* Write the image file to your SD card (e.g. with [Etcher](https://etcher.io)), depending on your environment you might need to extract the compressed image file first (e.g. with [WinRAR](http://www.rarlab.com/download.htm) or [7zip](http://www.7-zip.org))
+* Insert the SD card into the Pine A64, connect Ethernet and power ([See here for more details](http://wiki.pine64.org/index.php/Main_Page#Step_by_Step_Instructions))
+* Wait up to **15 minutes** for openHABian to do it's magic
+* Enjoy! 🎉
+
+* The device will be available under it's IP or via the local DNS name `openhabianpine64`
```

Comment:
Wow really... The difference is totally clear to me, why am I typing it wrong all the time ^^